src/_posts/platform/getting-started/2000-01-01-troubleshooting-ssh.md

```diff
@@ -20,10 +20,10 @@ When pushing to the platform, if you get the following error:
 
 ```bash
 $ git push scalingo master
-fatal: Could not read from remote repository.
-
-Please make sure you have the correct access rights
-and the repository exists.
+> fatal: Could not read from remote repository.
+>
+> Please make sure you have the correct access rights
+> and the repository exists.
```

Comment:
remark: I'm not sure I like these changes, I don't find it especially more readable that way. We don't have a doctrine to follow on this so that's just my 2 cents on this change. Not blocking at all.

Reply:
Ok I will update this